tests/test_redis_cluster_result_storage.py

```diff
@@ -0,0 +1,289 @@
+
+        topic = self.storage.last_updated()
+        expect(topic).not_to_be_null()
+        expect(topic).to_be_lesser_than(datetime.now())
+
+
+class CanReadUnexpiryLastUpdated(RedisDBContext):
+    def test_should_get_unexpire_last_updated(self):
+        self.connection.set("test_last_updated", IMAGE_BYTES)
+        self.ctx.request.max_age = 10
+        ttl = self.connection.ttl("test_last_updated")
+        topic = self.storage.last_updated()
+        expect(topic).not_to_be_null()
+        expect(ttl).to_equal(-1)
+
+
+class CanRaiseErrors(RedisDBContext):
+    @pytest.mark.asyncio
+    async def test_should_throw_an_exception(self):
+        config = Config(
+            REDIS_CLUSTER_RESULT_STORAGE_STARTUP_INSTANCES="localhost:6390,localhost:6391,localhost:6392",
+            REDIS_RESULT_STORAGE_MODE="cluster",
+            REDIS_RESULT_STORAGE_IGNORE_ERRORS=False,
+        )
+        ctx = Context(
+            config=config,
+            server=get_server("ACME-SEC"),
+        )
+        ctx.request = RequestParameters(
+            url=IMAGE_URL,
+        )
+        storage = RedisStorage(
+            context=ctx,
+            shared_client=False,
+        )
+
+        try:
+            topic = await storage.get()
+        except Exception as redis_error:
+            expect(redis_error).not_to_be_null()
+            expect(redis_error).to_be_an_error_like(redis.RedisError)
+
+
+class CanIgnoreErrors(RedisDBContext):
+    @pytest.mark.asyncio
+    async def test_should_not_throw_an_exception(self):
+        cfg = Config(
+            REDIS_CLUSTER_RESULT_STORAGE_STARTUP_INSTANCES="localhost:6390,localhost:6391,localhost:6392",
+            REDIS_RESULT_STORAGE_MODE="cluster",
+            REDIS_RESULT_STORAGE_IGNORE_ERRORS=True,
+        )
+        ctx = Context(
+            config=cfg,
+            server=get_server("ACME-SEC"),
+        )
+        ctx.request = RequestParameters(
+            url=IMAGE_URL % 10,
+        )
+        storage = RedisStorage(
+            context=ctx,
+            shared_client=False,
+        )
+
+        topic = await storage.get()
+        expect(topic).to_equal(None)
+        expect(topic).not_to_be_an_error()
+
+
+class RedisModeInvalid(RedisDBContext):
+    def setUp(self):
+        super().setUp()
+
+        self.cfg = Config(
+            REDIS_CLUSTER_RESULT_STORAGE_STARTUP_INSTANCES="localhost:6390,localhost:6391,localhost:6392",
+            REDIS_RESULT_STORAGE_MODE="test",
+        )
+        self.ctx = Context(
+            config=self.cfg,
+            server=get_server("ACME-SEC"),
+        )
+
+    @pytest.mark.asyncio
+    async def test_should_raises_attribute_error(self):
+        with self.assertRaises(AttributeError) as error:
+            RedisStorage(self.ctx)
+
+        expect(str(error.exception)).to_equal(
+            "Unknown value for REDIS_RESULT_STORAGE_MODE test. See README for more information."
+        )
+
+
+class ConnectToSentinelDatabase(RedisDBContext):
```

Comment:
Did you forget this sentinel test inside your file?!

Reply:
Oops... Removed ✔️ 
Waiting for approval from @jaspervanveghel...

Reply:
Thank you! Now LGTM! 🥳